geoips/commandline/geoips_command.py

```diff
@@ -64,14 +65,14 @@ class ParentParsers:
     shared correctly.
     """
 
-    geoips_parser = argparse.ArgumentParser(add_help=False)
+    geoips_parser = argparse.ArgumentParser()
     geoips_parser.add_argument(
-        "--log_level",
         "-log",
+        "--log-level",
         type=str,
         default="interactive",
-        choices=["debug", "error", "info", "interactive", "warning"],
-        help="The logging level to use for output via the CLI.",
+        choices=["interactive", "debug", "info", "warning", "error"],
```

Comment:
I think this line should be reverted to the previous order.

Reply:
It seems ``interactive`` is set to level 35. Debug is 10, info is 20, warning is 30, error is 40, critical is 50. So the list should actually be in the order of ``[debug, info, warning, interactive, error, critical]``.

Reply:
This has since been addressed.